spec/models/comment_spec.rb

```diff
@@ -24,6 +24,14 @@
 
       expect(comment.mentions).to eq [user1, user2]
     end
+
+    it 'detects admins in mentions' do
+      user1   = create(:user, :admin, email: 'admin@dradis.test')
+      user2   = create(:user, email: 'foo@dradis.test')
+      comment = create(:comment, content: 'Hello @admin@dradis.test and @foo@dradis.test')
+
+      expect(comment.mentions).to eq [user1, user2]
```

Comment:
in `CE` we are switching to `match_array`.
I am not sure why, but we should be consistent.

Reply:
I'm guessing it's due to the default ordering of the database, which does not exist. This means that `eq` will only work sometimes. `match_array` or `contains_exactly` should be the right method to use here since the order does not really matter.